### ISSUE TITLE
Trying to install 

### ISSUE THREAD

**Issue by decentropy** [hoytech/strfry#62]
I ran these

```
$ git clone https://github.com/hoytech/strfry.git
$ cd strfry
$ sudo apt install -y git build-essential libyaml-perl libtemplate-perl libregexp-grammars-perl libssl-dev zlib1g-dev liblmdb-dev libflatbuffers-dev libsecp256k1-dev libzstd-dev
$ git submodule update --init
$ make setup-golpe
$ make -j4
```

am getting this
`error: /root/bin/strfry/fbs/nostr-index.fbs:4: 16: error: expecting: ] instead got: :`

believe I have correct compiler installed
`gcc-10-base/focal-updates,focal-security,now 10.5.0-1ubuntu1~20.04 amd64 [installed]`

thanks if you have any ideas

**Comment by hoytech** [comment]
This error is from `flatc`. I think the version shipped with ubuntu 20.04 is too old for one of the features I'm using. Sorry!

It will take some more time, but I'm planning on phasing out that feature which should allow it to build on 20.04. BUT 20.04 also has an outdated libsecp256k library without schnorr signatures so linking will probably fail.

In short: If you can use 22.04 or higher, everything will be much easier!

**Comment by decentropy** [comment]
Thank you, I'll look into that option...
